metadata.yaml

```
@@ -181,6 +181,11 @@ spec:
         varType: bool
         defaultValue: false
     outputs:
+      - name: env_var
+        description: Map of environment variables for Spanner databases with keys as project_id:instance_name:database_name and values as fully qualified database paths
+        type:
+          - object
```

The generated output is inferring specific key for your deployed database as field in the type. This type field may not be available in some other deployment. I would recommend to manually override the type as below.

```suggestion
          - map
          - string
```